docker/cache/resources/repo-config.ttl

```diff
@@ -22,8 +22,12 @@
             owlim:enable-context-index "false" ;
 
             # indexes to speed up the read queries
-            owlim:enablePredicateList "true" ;
-            owlim:enable-literal-index "true" ;
-            owlim:in-memory-literal-properties "true" ;
+#            owlim:enablePredicateList "true" ;
+#            owlim:enable-literal-index "true" ;
+#            owlim:in-memory-literal-properties "true" ;
+            owlim:enablePredicateList "false" ;
+            owlim:enable-literal-index "false" ;
+            owlim:in-memory-literal-properties "false" ;
+
```

Comment:
Dont know enough about this stuff to comment really, but trust its working appropriately